test/stability/testSingleWorkingFF/testSingleWorkingFF.go

```diff
@@ -446,22 +450,23 @@ func generatePacket(pkt *packet.Packet, context flow.UserContext) {
 	atomic.AddUint64(&count, 1)
 }
 
-func checkInputFlow1(pkt *packet.Packet, context flow.UserContext) {
+func checkInputFlow1(pkt *packet.Packet, context flow.UserContext) bool {
 	if commonCheck(pkt) == false {
-		return
+		return false
 	}
 	udp := pkt.GetUDPForIPv4()
 	if udp.DstPort != packet.SwapBytesUint16(dstPort1) {
 		println("Unexpected packet in inputFlow1", udp.DstPort)
 		atomic.AddUint64(&brokenPackets, 1)
-		return
+		return false
```

Comment:
No, you can't use this function in Receive part. In Receive part you will have all packets at receive. It means that there will be packets with "udp.DstPort != packet.SwapBytesUint16(dstPort1)". Can you please run tests after these changes?

Reply:
Yes you are right. I changed receive side to call ShouldBeSkipped function instead.